compiler/tests/fixtures/features/return_sum_list_expression_1.fe

```diff
@@ -0,0 +1,8 @@
+contract Foo:
+
+    pub def bar() -> u256:
+        my_array: u256[3] = [10, 20, 30]
```

Comment:
what happens if we, say, use an array of size 20? :imp: 

Im wondering if we have stack problems during yul compilation.

Reply:
Oh yeah, I missed to check that...I'll check it.

Reply:
Updated the tests, appears to be working fine :sweat_smile:

Reply:
![bruns](https://user-images.githubusercontent.com/32600263/117201100-7c372880-ada9-11eb-823c-4eb98064586d.jpg)